speed/speed.js

```diff
@@ -557,29 +565,30 @@ function( Benchmark, document, selectors ) {
 			fastestHz = 0;
 			slowestHz = Infinity;
 			for ( i in scores ) {
-				if ( scores[i] > fastestHz ) {
-					fastestHz = scores[i];
+				if ( scores[ i ] > fastestHz ) {
+					fastestHz = scores[ i ];
 					fastest = i;
-				} else if ( scores[i] < slowestHz ) {
-					slowestHz = scores[i];
+				} else if ( scores[ i ] < slowestHz ) {
+					slowestHz = scores[ i ];
 					slowest = i;
 				}
+
 				// Format scores for display
-				scores[i] = Benchmark.formatNumber( scores[i].toFixed(2) ) + "o/s";
+				scores[ i ] = Benchmark.formatNumber( scores[ i ].toFixed( 2 ) ) + "o/s";
 			}
 
 			// Add conclusion to the header
-			elem = document.createElement("h3");
+			elem = document.createElement( "h3" );
 			elem.innerHTML = "The fastest is <i>" + fastest + " (" +
 				scores[ fastest ] + ")</i>. " +
 				"The slowest is <i>" + slowest + " (" +
 				scores[ slowest ] + ")</i>.";
-			get("header").appendChild( elem );
+			get( "header" ).appendChild( elem );
 
 			// Add totals to table
 			i = firstTestedColumn();
-			while ( (elem = tds[ i++ ]) ) {
-				attr = elem.getAttribute("data-engine");
+			while ( ( elem = tds[ i++ ] ) ) {
```

Comment:
@gibson042 distinguishing grouping parentheses with other cases is pretty hard goal to accomplish :-(, otherwise i would did it a long time.

Nevertheless, you are very welcome to take a stab on it yourself